ostis-apps/nika
# Feature/message about entity

To test:
- Type "Что такое <ваша сущность>", see that there is no answer
- Create construction like in the screenshot using scg-editor. You can use `kb/pattern/instance` pattern. Use `nrel_note` or `nrel_definition`
- Don't forget to add new entity to `concept_wit_entity` class
- Type "Что такое <ваша сущность>" again and see the answer
![image](https://github.com/ostis-apps/nika/assets/36566977/76d32320-13a5-4bba-9c6d-0222b3c2df03)
